### PR TITLE
Adds UIUC fleet fuel consumption analysis

### LINKED ISSUE #10: Add UIUC fleet fuel consumption and its respective analysis
This issue can be closed when the following tasks are completed: 
- Add to the repo the fuel consumption annual data of UIUC Facilities and Services fleet.
- Add a script to analyze such data.

### PR DESCRIPTION
**Pull request by robfairh**
Closes #10

### CODE REVIEW

File: calculate.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    --------
+
+    """
+    fig, ax = plt.subplots(figsize=(15, 7))
+    ax.plot(fuel)
+    ax.set_title('Fuel Consumption')
+    ax.grid()
+    ax.set(xlim=(0, 364), ylim=(0, 850))
+    ax.set_xlabel("Days", fontsize=14)
+    ax.set_ylabel("Quantity (Gallons)", fontsize=14)
```
Comment (katyhuff) [inline calculate.py]: Something more specific here. 
```suggestion
    ax.set_ylabel(r'Fuel Consumed $\left(\frac{gal}{d}\right)$', fontsize=14)
```

File: calculate.py [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+        list of values to be plotted. First value corresponds to the
+        beginning of the year.
+
+    Returns:
+    --------
+
+    """
+    fig, ax = plt.subplots(figsize=(15, 7))
+    ax.plot(fuel)
+    ax.set_title('Fuel Consumption')
```
Comment (katyhuff) [inline calculate.py]: Something more specific here.

```suggestion
    ax.set_title('Daily {0} Fuel Consumption by the UIUC fleet'.format(fuel.capitalize()))
```

**Review by katyhuff — CHANGES_REQUESTED** [review]

File: calculate.py [hunk 3]
```diff
@@ -0,0 +1,96 @@
+""" This script analyses fuel consumption in a year.
```
Comment (samgdotson) [inline calculate.py]:
```suggestion
""" 
This script analyses fuel consumption in a year.
```

File: calculate.py [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+- Average fuel consumed (total fuel consumed/365).
+- Plot of the fuel consumed throughout the year.
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def plotfuel(fuel):
+    """ Plots 'fuel'.
```
Comment (samgdotson) [inline calculate.py]:
```suggestion
    """ 
    Plots 'fuel'.
```

**Review by samgdotson — CHANGES_REQUESTED** [review]
Very minor style changes. You could run ``autopep8`` and it will format for you.

File: calculate.py (hunk 3 above)
Comment (robfairh) [reply calculate.py]: Cool, thanks!

**Comment by robfairh** [comment]
I committed both of your suggestions. Thank you!

**Review by samgdotson — APPROVED** [review]
Looks good

**Pull request merged**